README.md

```diff
@@ -0,0 +1,59 @@
+# CENNZnet Node
+[![GitHub license](https://img.shields.io/github/license/cennznet/cennznet)](LICENSE) [![CircleCI](https://circleci.com/gh/cennznet/cennznet.svg?style=shield)](https://circleci.com/gh/cennznet/cennznet) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](docs/CONTRIBUTING.adoc)
+
+CENNZnet node based on [Plug](https://github.com/plugblockchain/plug-blockchain)
+
+## Running a CENNZnet node
+
+There are a number of ways to run a CENNZnet node. Please choose one that suits best for your interest.
+
+### Using Docker
+
+Make sure Docker is installed and running on your machine.
+```
+$ docker pull cennznet/cennznet:1.0.0
+$ docker run -it cennznet/cennznet:1.0.0 [TODO: check params]
```

Comment:
`pull` is handled automatically these days when trying to run.
Let's set this to the latest available image `1.0.0-rc2`

```
# Start a local validator on a development chain
docker run \
    -p 9933:9933 -p 9944:9944 \
    cennznet/cennznet:1.0.0-rc2 --dev --unsafe-ws-external --unsafe-rpc-external
```